custom/templates/DefaultRevamp/css/custom.css

```
@@ -1486,6 +1501,27 @@ body.dark .cc-window.cc-floating .cc-highlight .cc-btn.cc-allow {
     border-color: transparent;
 }
 
+body.dark #modal-reactions .ui.menu {
+    background-color: #282828;
+    overflow-x: auto;
```

```suggestion
    overflow: auto;
```
Sorry, meant reactions as in reaction types but also reactions as in reactions given by users 😅 so both directions need this

---

The `scrolling content` class on the outer div should do this already 👍